common/sgx/tcbinfo.h

```diff
@@ -13,23 +13,58 @@
 
 OE_EXTERNC_BEGIN
 
-typedef enum _oe_tcb_level_status
-{
-    OE_TCB_LEVEL_STATUS_UNKNOWN,
-    OE_TCB_LEVEL_STATUS_REVOKED,
-    OE_TCB_LEVEL_STATUS_OUT_OF_DATE,
-    OE_TCB_LEVEL_STATUS_CONFIGURATION_NEEDED,
-    OE_TCB_LEVEL_STATUS_UP_TO_DATE,
-    __OE_TCB_LEVEL_MAX = OE_ENUM_MAX,
+#define OE_TCB_LEVEL_STATUS_UNKNOWN (0)
+#define OE_TCB_LEVEL_STATUS_REVOKED (1 << 0)
```

Comment:
Are lines 17-21 used by anything?   Would be better to delete them.

Reply:
Just there for reference, but I agree is a little redundant.  Will remove.